src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -92,10 +96,12 @@ class ApplicationErrorBoundary extends React.Component {
       const errorDetails = activeError.message.toString();
       const errorText = intl.formatMessage({ id: 'terraApplication.errorBoundary.defaultErrorMessage' }, { errorDetails });
       return (
-        <StatusView
-          variant="error"
-          message={errorText}
-        />
+        <div ref={this.errorBoundaryRef} role="alert" tabIndex="-1">
```

Comment:
Is the "alert" role is appropriate here? It does ensure that the error text is "assertive" and that it's read when presented, but I'm not sure if there are other side effects.